test/components/midpoint_test.dart

```diff
@@ -20,8 +29,139 @@ main() {
         ),
       ),
     );
-    print(result.coordinates.join(', '));
-    expect(_lngRange(result.coordinates.lng), true);
-    expect(_latRange(result.coordinates.lat), true);
+    checkLatLngInRange(result);
+  });
+
+  test('midpoint -- horizontal equator', () {
+    Point pt1 = Point(
+      coordinates: Position.named(
+        lng: 0,
+        lat: 0,
+      ),
+    );
+    Point pt2 = Point(
+      coordinates: Position.named(
+        lng: 10,
+        lat: 0,
+      ),
+    );
+    Point result = midpoint(pt1, pt2);
+
+    checkLatLngInRange(result);
```

Comment:
I think this test is actually missing the distance check that the other tests have, which also perhaps could be pulled out into a separate function, like `checkLatLngInRange`. I'll keep the fix on hold in case there is other feedback that should be addressed in another patch.